assets/src/scripts/components/SearchBar/SearchBarResult.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+/**
+ * History List.
+ */
+export default class SearchBar_Result extends React.Component {
+
+	render() {
+
+		var classes = [ 'SearchBar_Result' ];
```

Comment:
I suppose we can take the approach of making everything `let` by default and only giving it greater permission when needed. 

But I'm not so sure in this case. I'm concerned using `let` signals that it **shouldn't** be modified from outside the block of code in future. EG you shouldn't modify `classes` from within a callback or if statement? For this particular case, I wouldn't want to imply that shouldn't be done?